File: app/src/main/java/com/adisa/diningplus/fragments/FollowDialogFragment.java
package com.adisa.diningplus.fragments;

import android.app.Dialog;
import android.content.DialogInterface;
import android.os.Bundle;
import androidx.appcompat.app.AlertDialog;
import androidx.fragment.app.DialogFragment;

/**
 * Created by Adisa on 5/3/2017.
 */

public class FollowDialogFragment extends DialogFragment {
    public Dialog onCreateDialog(final Bundle savedInstanceState) {
        // Use the Builder class for convenient dialog construction
        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
        builder.setTitle("Following menu items")
                .setMessage("Click on the notification button to be notified when this menu item is being served again.")
                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
                    public void onClick(DialogInterface dialog, int id) {

                    }
                });
        // Create the AlertDialog object and return it
        return builder.create();
    }
}

File: app/src/main/java/com/adisa/diningplus/fragments/FollowDialogFragment.kt
package com.adisa.diningplus.fragments

import android.app.Dialog
import android.os.Bundle
import androidx.appcompat.app.AlertDialog
import androidx.fragment.app.DialogFragment

class FollowDialogFragment : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        // Use the Builder class for convenient dialog construction
        val builder = AlertDialog.Builder(activity!!)
        builder.setTitle("Following menu items")
                .setMessage("Click on the notification button to be notified when this menu item is being served again.")
                .setPositiveButton("OK") { dialog, id -> }
        // Create the AlertDialog object and return it
        return builder.create()
    }
}